### PR TITLE
CD compatible changes

### PR DESCRIPTION
**Pull request by Satendra-SR**
### Changes include:

- CD compatible changes. added Dockerfile for dockerized env.
- Need to figure out how we can pass the env variable on RUN time in react. For now, added fallback hardcoded url.

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
 export const DATE_FORMAT = 'DD/MM/YY';
 export const TIME_FORMAT = 'HH:mm';
-export const REACT_APP_GLIFIC_REGISTRATION_API =
-  process.env.REACT_APP_GLIFIC_API + '/v1/registration';
+export const REACT_APP_GLIFIC_REGISTRATION_API = process.env.REACT_APP_GLIFIC_API ? process.env.REACT_APP_GLIFIC_API + '/v1/registration' : 'https://api.glificuat.coloredcow.com/api/v1/registration';
 export const REACT_APP_GLIFIC_AUTHENTICATION_API =
-  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send-otp';
-export const USER_SESSION = process.env.REACT_APP_GLIFIC_API + '/v1/session';
-export const RESET_PASSWORD = process.env.REACT_APP_GLIFIC_API + '/v1/registration/reset-password';
+  process.env.REACT_APP_GLIFIC_API ?  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send-otp': 'https://api.glificuat.coloredcow.com/api/v1/registration/send-otp';
```
Comment (kurund) [inline src/common/constants.ts:6]: We can't hardcode URLs here.

File: src/config/index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const envVariables = process.env;
 
-export const URI = envVariables.REACT_APP_GLIFIC_API;
-export const SOCKET = envVariables.REACT_APP_WEB_SOCKET;
+export const URI = envVariables.REACT_APP_GLIFIC_API  ? envVariables.REACT_APP_GLIFIC_API : 'https://api.glificuat.coloredcow.com/api' ;
+export const SOCKET = envVariables && envVariables.REACT_APP_WEB_SOCKET ? envVariables.REACT_APP_WEB_SOCKET : 'wss://api.glificuat.coloredcow.com/socket';
 export const SENTRY_DSN = envVariables.SENTRY_DSN;
-export const FLOW_EDITOR_API = envVariables.REACT_APP_FLOW_EDITOR_API;
+export const FLOW_EDITOR_API = envVariables && envVariables.REACT_APP_FLOW_EDITOR_API ? envVariables.REACT_APP_FLOW_EDITOR_API : 'https://api.glificuat.coloredcow.com/flow-editor/';
```
Comment (kurund) [inline src/config/index.ts:6]: We can't hard code URLs hers.

File: src/common/constants.ts (hunk 1 above)
Comment (Satendra-SR) [reply src/common/constants.ts]: I agree @kurund that was just a workaround for testing CD. I've figured out about the env variables now and reverted these changes now.

**Review by kurund — APPROVED** [review]
Looks good.

**Pull request merged**